ister_gui.py

```diff
@@ -1110,6 +1114,11 @@ def update_widgets(self):
         if not dns_res:
             dns_res = 'none found'
 
+        if 'none found' in ''.join([interface_res, mask_res, ip_res, dns_res]):
```

Comment:
set_text is is adding the refresh message to the header iff 'none found' was in a result, which is almost always because there was not enough time to set it.

The alternative is to wait until everything is found, but most people won't be needing to set static IP for the installer so why wait for that?